Compiler/src/Compiler.cpp

```diff
@@ -2478,12 +2478,13 @@ struct Compiler
         }
         else if (AstStatContinue* stat = node->as<AstStatContinue>())
         {
+            LUAU_ASSERT(!loops.empty());
```

Comment:
nit: might make sense to move the assert in AstStatBreak as well, this is why this issue was here in the first place - the code block was copied from Break to Continue and then extra code was added before the comment.